third_party/binutils/test_decenc/test_decenc_x86.asm

```diff
@@ -139588,6 +139588,18 @@ GLOBAL_LABEL(FUNCNAME:)
         RAW(7b) RAW(00) RAW(00) RAW(00)
         RAW(62) RAW(f2) RAW(7d) RAW(29) RAW(91) RAW(b4) RAW(f5)
         RAW(7b) RAW(00) RAW(00) RAW(00)
+        END_OF_SUBTEST_MARKER
+
+        /* xsaves.s */
```

Comment:
What about the disabled existing test of this which needs editing:

```
third_party/binutils/test_decenc/test_decenc_x86.asm-        /* xsaves, CPL 0 instruction. */
third_party/binutils/test_decenc/test_decenc_x86.asm:        /* RAW(0f) RAW(c7) RAW(29) */
--
third_party/binutils/test_decenc/test_decenc_x86_64.asm-        /* xsaves, CPL 0 instruction. */
third_party/binutils/test_decenc/test_decenc_x86_64.asm:        /* RAW(0f) RAW(c7) RAW(29) */
```

Reply:
Deleted that. That one disabled test is already a part of the new ones I added (with a different reg).